Source/JavaScriptCore/JavaScriptCore.xcodeproj/project.pbxproj

```diff
@@ -11927,6 +11924,28 @@
 			shellPath = /bin/sh;
 			shellScript = "set -e\n\nif [ \"${ACTION}\" = \"installhdrs\" ]; then\n    exit 0;\nfi\n\nif [ -f ../../Tools/Scripts/check-for-inappropriate-macros-in-external-headers ]; then\n    ../../Tools/Scripts/check-for-inappropriate-macros-in-external-headers Headers PrivateHeaders/JSBasePrivate.h\nfi\n";
 		};
+		DD9C4CC52A05D4FB00D52E2E /* Fix up SDKROOT path in TAPI filelists (install only) */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 8;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+				"$(SDKROOT)$(WK_LIBRARY_HEADERS_FOLDER_PATH)/bmalloc/bmalloc.json",
+				"$(SDKROOT)$(WK_LIBRARY_HEADERS_FOLDER_PATH)/wtf/WTF.json",
+			);
+			name = "Fix up SDKROOT path in TAPI filelists (install only)";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+				"$(DERIVED_FILE_DIR)/bmalloc.json",
+				"$(DERIVED_FILE_DIR)/WTF.json",
+			);
+			runOnlyForDeploymentPostprocessing = 1;
+			shellPath = /bin/sh;
+			shellScript = "for ((i=0; i<${SCRIPT_INPUT_FILE_COUNT}; i++)); do\n    eval input=\"\\${SCRIPT_INPUT_FILE_$i}\"\n    eval output=\"\\${SCRIPT_OUTPUT_FILE_$i}\"\n    sed -e \"s;<SDKROOT>;${SDKROOT};\" $input > $output\ndone\n";
```

Comment:
Calling attention to more change I discovered while doing internal testing.

> In installhdrs builds of WTF and bmalloc, all-products-headers.yaml is
> not available, so the script produces its file list by scanning the
> headers directory being installed. This doesn't carry the staleness risk
> of scanning directories in normal builds, because install-style builds
> are always clean.

Here's the script expanded:
```sh
for ((i=0; i<${SCRIPT_INPUT_FILE_COUNT}; i++)); do
    eval input="\${SCRIPT_INPUT_FILE_$i}"
    eval output="\${SCRIPT_OUTPUT_FILE_$i}"
    sed -e "s;<SDKROOT>;${SDKROOT};" $input > $output
done
```